R/list-combine.R

```diff
@@ -36,27 +37,31 @@
 #' list_rbind(unname(x2), names_to = "id")
 #'
 #' list_cbind(x2)
-list_c <- function(x, ptype = NULL) {
+list_c <- function(x, ..., ptype = NULL) {
   vec_check_list(x)
+  check_dots_empty()
```

Comment:
This is like the most mild of comments, but I always make `check_dots_empty()` the _very first_ thing my functions do. Somehow that shows that it is a technical check rather than more meaningful check like `check_bool()` or `check_list_of_data_frames()`

Reply:
IMO it's simpler to check the arguments in the order they appear, with no special cases.